tests/rhosak/kas_messaging.spec.ts

```diff
@@ -96,19 +96,20 @@ test.beforeEach(async ({ page }) => {
   expect(producerResponse === true).toBeTruthy();
 });
 
-test.afterAll(async ({ page }) => {
-  await navigateToKafkaList(page);
-
+test.afterEach(async ({ page }) => {
+  await navigateToSAList(page);
   try {
-    await deleteKafkaInstance(page, testInstanceName);
+    await deleteServiceAccount(page, testServiceAccountName);
```

Comment:
this is already in beforeEach, what is the difference as you keep both logic?

Reply:
we have to clean up after, the one that is on beforeEach it is supposedly cleaning the one using the same name, which does not make sense anymore as we are using session id and the name will differ
```
if ((await page.locator('tr', { hasText: testServiceAccountName }).count()) !== 0) {
    await deleteServiceAccount(page, testServiceAccountName);
  }
```
I think I should delete those lines

Reply:
Can I resolve this conversation @Frawless ?